app/assets/javascripts/govuk_publishing_components/analytics/page-content.js

```diff
@@ -71,7 +71,7 @@
   function getMetaAttribute (selector) {
     var element = document.querySelector(selector)
     if (element) {
-      return element.getAttribute('content')
+      return element.getAttribute('content').toLowerCase()
```

Comment:
👍

Reply:
Couldn't figure out why this was working on live but not local. Still weird to me that live is `collections` but local is `Collections`...